tutorials/appstudio-onboarding/appstudio-onboarding.md

```diff
@@ -35,21 +35,25 @@ Go to <https://account.hanatrial.ondemand.com> and log in to your SAP Cloud Plat
 
     >In recently created trial accounts, SAP Business Application Studio is subscribed to by default.
 
-    >If you receive an error message and your recently created account was created with error, it is recommended that you re-create your account: delete the account and then [Get a Free Trial Account on SAP Cloud Platform](hcp-create-trial-account).
+    >If you receive an error message and your recently created account was created with error, it is recommended that you re-create your account: delete the account and then [Get a Free Account on SAP BTP Trial](hcp-create-trial-account).
 
     >If you receive an error message and your account was not created recently, answer the validation question of the current step, go to the next step in this tutorial, and proceed from there.
 
 2. You might be asked to accept the legal terms. Check the box and click **OK**.
 
     !![legalterms](2020-03 AppStudio Terms__.jpg)
 
-3. A new tab opens and SAP Business Application Studio loads.
+3. A new tab opens and the welcome page for SAP Business Application Studio loads.
 
+!![sapbusinessapplicationstudiowelcome](BAS_Welcome_.png)
```

Comment:
Please follow the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64) when naming your images.